os/arch/arm/src/common/up_usestack.c

```diff
@@ -134,9 +134,9 @@ int up_use_stack(struct tcb_s *tcb, void *stack, size_t stack_size)
 	 */
 
 #ifdef CONFIG_ARCH_ARMV7A_FAMILY
-	top_of_stack = (uint32_t)tcb->stack_alloc_ptr + stack_size;
+	top_of_stack = (uint32_t) tcb->stack_alloc_ptr + stack_size;
```

Comment:
same as previous. 
Please check others as well.

Reply:
I will check and correct it.